edge/discussion_materials/ONNX Edge scope and profile definition.md

```diff
@@ -0,0 +1,72 @@
+# ONNX Edge scope and profile definition
+Author: Yedong Liu, Milan Oljaca, Saurabh Tangri
+
+## Scope of Edge
+
+Edge computing is computing that’s done at or near the source of the data, instead of relying on the cloud at one of a dozen data centers to do all the work. Edge computation is largely or completely performed on distributed edge devices. The target of edge computing is any application or general functionality needing to be closer to the source of the action where system can process and interact with the real world instantly.
+
+
+## Edge scenario
+
+Computation offloading for real-time applications, such as facial recognition, smart city, intelligent traffic control, industrial IoT, video analysis and monitoring, border control, unattended shops, auto pilot and smart drones are all considered within edge computing scope. 
```

Comment:
edge computing goes beyond computation offload - it may also be due to privacy, limited connectivity, latency requirement etc. I suggest to rephrase this section similar to the following:
"Doing the computing at the edge may be due to various reasons - need for real-time, privacy, latency, connectivity and more. A few examples for edge usage scenarios there are: face recognition, smart assistant, smart city, intelligent traffic control, industrial IoT, video analysis and monitoring, border control, unattended shops, autonomous cars and smart drones."